pkg/sql/logictest/testdata/logic_test/sequences

```diff
@@ -133,9 +133,41 @@ TRUNCATE foo
 statement error pgcode 42809 "foo" is not a table
 DROP TABLE foo
 
-# TODO(vilterp): match PG (returns a single row with the sequence settings and value)
-statement error pgcode 42809 pq: cannot SELECT from a sequence
-SELECT * FROM foo
+# You can select from a sequence to get its value.
+
+statement ok
+CREATE SEQUENCE select_test
+
+query IIB colnames
+SELECT * FROM select_test
+----
+last_value log_cnt is_called
+0          0       true
```

Comment:
@vivekmenezes: the `is_called` and `log_cnt` are always returned as 0 and true, respectively, because they're meaningless in CR — we don't store an `is_called` flag or a `log_cnt` value (log_cnt has something to do with Postgres writing to its WAL log). Maybe we should just not return these columns altogether? I just put them there for PG compatibility, although PG doesn't even document them.

cc @lhirata